reporter/src/funTest/kotlin/reporters/ExcelReporterFunTest.kt

```diff
@@ -52,25 +53,35 @@ class ExcelReporterFunTest : WordSpec({
             val expectedSheetNames = expectedWorkbook.sheetIterator().asSequence().map { it.sheetName }.toList()
 
             actualSheetNames shouldContainExactly expectedSheetNames
-            actualSheetNames.map { "Sheet" to it }.forAll { (_, sheetName) ->
-                val actualSheet = actualWorkbook.getSheet(sheetName)
-                val actualRowNums = actualSheet.uniqueRowIterator().map { it.rowNum }.toList()
+            actualSheetNames.map { Address(it) }.forAll { sheetAddress ->
+                val actualSheet = actualWorkbook.getSheet(sheetAddress.sheet)
+                val actualRowNums = actualSheet.uniqueRowIterator().map {
+                    sheetAddress.copy(row = it.rowNum)
+                }.toList()
 
-                val expectedSheet = expectedWorkbook.getSheet(sheetName)
-                val expectedRowNums = expectedSheet.uniqueRowIterator().map { it.rowNum }.toList()
+                val expectedSheet = expectedWorkbook.getSheet(sheetAddress.sheet)
+                val expectedRowNums = expectedSheet.uniqueRowIterator().map {
+                    sheetAddress.copy(row = it.rowNum)
+                }.toList()
 
                 actualRowNums shouldContainExactly expectedRowNums
-                actualRowNums.map { "Row" to it }.forAll { (_, rowNum) ->
-                    val actualRow = actualSheet.getRow(rowNum)
-                    val actualColumnIndices = actualSheet.uniqueCellIterator(actualRow).map { it.columnIndex }.toList()
-
-                    val expectedRow = expectedSheet.getRow(rowNum)
-                    val expectedColumnIndices = actualSheet.uniqueCellIterator(expectedRow).map { it.columnIndex }.toList()
+                actualRowNums.forAll { rowAddress ->
+                    // The non-null assertion is fine as we pass "it" as a row directly above.
```

Comment:
Which "it" do you refer to?

Reply:
Ah, sorry, that is a left-over from a previous iteration. Will fix.